resources/views/components/accordion/mobile.blade.php

```diff
@@ -0,0 +1,25 @@
+{{--
+This is a responsive version of the accordion component.
+
+Responsive behavior adjustments:
+    - The `label` slot applies `md:cursor-auto`,
+      making it non-clickable on larger screens,
```

Comment:
since when does cursor-auto make things non-clickable? isn't that pointer-events? and with md: it makes it clickable on larger screens